src/components/Amount.tsx

```diff
@@ -32,7 +32,7 @@ export default function Amount({
   }
```

Comment:
<img width="584" alt="image" src="https://github.com/user-attachments/assets/1aa3e5ca-f2e5-4ab0-9818-e6d839a4802c">

font too small, should be 26px not 22px